src/cloud/project/project-patch.md

```diff
@@ -1,51 +1,56 @@
 ---
 group: cloud-guide
-title: Apply custom patches
+title: Apply patches
 functional_areas:
   - Cloud
   - Upgrade
 ---
+The [Magento Cloud Patches](https://github.com/magento/magento-cloud-patches) package is a set of patches, previously used within the ece-tools package to improve the integration of all Magento versions with Cloud environments and to deliver critical fixes quickly. This can also be integrated to an existing Magento project as a standalone package.
 
-Sometimes we provide a [custom patch]({{ site.baseurl }}/guides/v2.3/comp-mgr/patching.html#custom-patches) to address a specific issue. Also, third-party extension developers can provide a custom patch. Copy the custom patch to the `/m2-hotfixes` directory and test it on your local workstation.
+Magento Cloud Patches can also apply [custom patches]({{ site.baseurl }}/guides/v2.3/comp-mgr/patching.html#custom-patches) provided by support or third-party extension developers. Copy the custom patch to a directory called `/m2-hotfixes` in the project root and test it on your local workstation.
 
 {% include cloud/note-upgrade.md %}
 
 {:.procedure}
-To apply and test a custom patch:
-
-You can only apply patches during the build phase of redeployment.
+To implement Magento Cloud Patches:
 
-1. On your local workstation, create a branch based on the `integration` branch.
+1. Add the Magento Cloud Patches package to your `composer.json`
 
     ```bash
-    magento-cloud environment:branch <branch-name>
+    composer require magento/magento-cloud-patches
     ```
+    
+{:.procedure}
+To apply patches for testing:
 
-1. Copy the patch file to the `/m2-hotfixes` directory.
+1. From the project root, apply the patches.
 
-1. Add, commit, and push your code changes.
+    ```bash
+     php ./vendor/bin/ece-patches apply
+    ```
+
+1. Clear the Magento cache.
 
     ```bash
-    git add -A && git commit -m "Apply patch" && git push origin <branch name>
+    php ./bin/magento cache:clean
     ```
 
-1. After test validation, merge this branch with the `integration` branch.
+    You can also clean the cache using the [Magento Admin Cache Management](http://docs.magento.com/m2/ee/user_guide/system/cache-management.html).
+
+1. Test the patches, make any necessary changes to custom patches.
```

Comment:
So for Magento-provided patches, you can test them in your local environment using these instructions, but if there are any issues with the patches, wouldn't you need to contact Magento?

Not clear on how patches work.  

Case 1:
Are patches automatically applied by ece-tools when you deploy your project?  So if you we release a new version of ece-tools, the customer upgrades, then any new patches would be applied automatically  when they deploy their project, or 

Case 2:
Customer has to add the Cloud patches command to their composer file, and then run the `apply patches` command manually.